### ISSUE TITLE
On logout with redirections is not preserved the /login?key=123 and redirects with the ?key=123

### ISSUE THREAD

**Issue by dxdemetriou** [Ylianst/MeshCentral#4420]
**Describe the bug**
When loginKey is used with Redirects, on Logout is not preserved the /login?key=123, just the ?key=123

**To Reproduce**
1. Go to 'https://[meshcentral.domain.com]/login?key=123'
2. Click on 'Logout'
3. You are redirected to 'https://[redirection.domain.com]/?key=123'

**Expected behavior**
After a logged-on user is logged-out should be redirected to the 'https://[meshcentral.domain.com]/login?key=123' instead of the 'https://[redirection.domain.com]/?key=123'

**Server Software (please complete the following information):**
 - OS: RockyLinux 8
 - Virtualization: [e.g. Docker]
 - Network: Hybrid
 - Version: 1.0.70
 - Node: v16.14.0

**Additional context**

**Your config.json file**
```
"ipBlockedUserRedirect": "https://...",
...
"rootRedirect": "https://...",
"unknownUserRootRedirect": "https://...",
"loginKey": ["123", "key2"],
```

**Comment by Ylianst** [comment]
Your right. I get it now. Working on this one now.

**Comment by Ylianst** [comment]
Fixed, will be in v1.0.71

**Comment by dxdemetriou** [comment]
Tried now, It works on desktop edition but not on mobile' website that keeps only the/login without the ?key

**Comment by Ylianst** [comment]
Doh! Ok. I need to look at the mobile page.

**Comment by Ylianst** [comment]
Fixed, will be in v1.0.72

**Comment by dxdemetriou** [comment]
Perfect, it works everywhere now. Thanks :smiley:
